packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMUtils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+#import "messages.g.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FGMUtils : NSObject
```

Comment:
I strongly discourage "classes" that are just bags of class methods; Objective-C, unlike Java, has a perfectly good way of representing functions that aren't part of a class, which is functions :)

Reply:
My original idea was to use the class for namespacing, but I have now added the functions as global function with FGM prefix.